chain/chain/src/store.rs

```diff
@@ -2077,7 +2077,11 @@ impl<'a> ChainStoreUpdate<'a> {
             self.gc_col(ColIncomingReceipts, &block_shard_id);
             self.gc_col(ColChunkPerHeightShard, &block_shard_id);
             self.gc_col(ColNextBlockWithNewChunk, &block_shard_id);
-            self.gc_col(ColChunkExtra, &block_shard_id);
+
+            // TODO: use the real shard version
```

Comment:
is there an issue for this?

Reply:
What do you mean?

I created an issue for this TODO https://github.com/near/nearcore/issues/4710. I will implement that in a later PR.

One issue is that state validator will fail if these columns are not garbage collected correctly. One way to hack that is to garbage collect both `shard_version` 0 and 1. The most correct way is pass the correct shard layout version to the gc function.

Reply:
Could we add the issue number here?